specification/draft/apps.mdx

```diff
@@ -450,13 +454,64 @@ Example:
     "hostInfo": { "name": "claude-desktop", "version": "1.0.0" },
     "hostContext": {
       "theme": "dark",
+      "styles": {
+        "--color-background-primary": "light-dark(#ffffff, #171717)",
+        "--color-text-primary": "light-dark(#171717, #fafafa)",
+        "--font-family-sans": "system-ui, sans-serif"
+      },
       "displayMode": "inline",
       "viewport": { "width": 400, "height": 300 }
     }
   }
 }
 ```
 
+### Theming
+
+Hosts pass CSS custom properties via `HostContext.styles` for visual cohesion with the host environment.
+
+#### Standardized Variables
+
+| Category | Variables |
+|----------|-----------|
+| Background | `--color-background-primary`, `--color-background-secondary`, `--color-background-tertiary`, `--color-background-inverted` |
+| Text | `--color-text-primary`, `--color-text-secondary`, `--color-text-tertiary`, `--color-text-inverted` |
+| Icons | `--color-icon-primary`, `--color-icon-secondary`, `--color-icon-tertiary`, `--color-icon-inverted` |
+| Borders | `--color-border-primary`, `--color-border-secondary` |
+| Accents | `--color-accent-info`, `--color-accent-danger`, `--color-accent-success`, `--color-accent-warning` |
+| Font Family | `--font-family-sans` |
+| Font Sizes | `--font-size-heading`, `--font-size-body`, `--font-size-caption` |
+| Font Weights | `--font-weight-regular`, `--font-weight-emphasized` |
+| Line Heights | `--font-leading-regular`, `--font-leading-tight` |
+| Composite Styles | `--font-style-heading`, `--font-style-body`, `--font-style-body-emphasized`, `--font-style-caption`, `--font-style-caption-emphasized` |
+| Border Radius | `--border-radius-small`, `--border-radius-medium`, `--border-radius-large`, `--border-radius-full` |
+| Border Width | `--border-width-regular` |
+
+#### Host Behavior
+
+- Hosts MAY provide any subset of standardized variables
+- Hosts MAY use CSS `light-dark()` function for theme-aware values
+- `theme` indicates the active mode; `styles` provides the concrete CSS values
+
+#### App Behavior
+
+- Apps SHOULD use fallback values for CSS variables: `var(--color-text-primary, #171717)`
```

Comment:
This is repetitive with the above (line 440-442). And I also have the same comment here as there